client/webserver/site/src/js/mmutil.ts

```diff
@@ -1008,21 +1011,31 @@ export function feesAndCommit (
     if (!baseIsAccountLocker && quoteFeeID !== baseFeeID) quoteRedeemReservesPerLot = quoteFees.max.redeem
   }
 
+  const baseReservesFactor = 1 + baseOrderReservesFactor
+  const quoteReservesFactor = 1 + quoteOrderReservesFactor
+
+  const baseBookingFees = (baseBookingFeesPerLot * baseLots) * baseReservesFactor
+  const baseRedeemFees = (baseRedeemReservesPerLot * quoteLots) * quoteReservesFactor
+  const quoteBookingFees = (quoteBookingFeesPerLot * quoteLots) * quoteReservesFactor
+  const quoteRedeemFees = (quoteRedeemReservesPerLot * baseLots) * baseReservesFactor
```

Comment:
I'm scaling the booking fees to reserve for the order reserves as well, because the fees reserved for an order are not necessarily released as soon as the order matches. But for the maker, the fees reserved for the swap actually should be released immediately, since the maker should broadcast its swap right away. And since for arb-mm, we usually expect to be the maker, we should maybe only scale the fees reserved for redemptions. I'm gonna leave it for now though, since it's not actually impossible that we'd be the taker.